resources/migrations/000_migrations.yaml

```diff
@@ -3828,88 +3828,4 @@ databaseChangeLog:
       - dropForeignKeyConstraint:
           baseTableName: raw_table
           constraintName: fk_rawtable_ref_database
-          remarks: 'This FK prevents deleting databases even though RAW_TABLE is no longer used. The table is still around to support downgrades, but the FK reference is no longer needed.'
```

Comment:
Wouldn't a [Liquibase rollback](http://www.liquibase.org/documentation/rollback.html)  be safer (say _someone_ 😊 *blush* already deployed RC2) - or what will otherwise happen if you guys reuse `changeset id: 65` to _someone's_ DATABASECHANGELOG?

Reply:
It's probably most bulletproof if you were to 
`DELETE FROM migrationchangelog WHERE id = '65'`
and drop the tables in question (`computation_job` and `computation_job_result`)

Reply:
actually we talked it out and @senior is going to add a cleanup migration with a precondition. If you can test it for us, that would be awesome!

Reply:
Sweet! Yes for sure we’ll do that tomorrow (on CEST time here - so with a bit of luck we should be able to work *with* the sun)

Thanks a bunch guys! Potentially this could save someone other than us too.